docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst

```diff
@@ -55,175 +58,154 @@ to start the daemon.
+the executable file to be used as well as some metadata.
+Let's have a look at the codes already installed on your machine:
 
-    code = load_code("<codename>")
+.. code:: console
+
+    verdi code list
 
-Here ``load_code`` (imported for you by ``verdi shell`` or the
-``verdi run`` command) returns a ``Code`` which is the general AiiDA
-class handling all possible codes, and ``code`` is an instance labeled
-as ``<codename>`` (see the first part of the tutorial for listing all
-codes installed in your AiiDA machine). You might also want to list only
-the codes that define a default calculation plugin for the pw.x code of
-Quantum ESPRESSO. You can do this with the following command:
+There should be a number of them. Here, we're interested in the "PWscf" executable of Quantum Espresso, i.e. in codes for the ``quantumespresso.pw`` plugin:
 
 .. code:: console
 
     verdi code list -P quantumespresso.pw
 
 Pick the correct codename, that might look like, e.g.
-``qe-6.3-pw@localhost``.
+``qe-6.3-pw@localhost`` and load it in the verdi shell.
 
-Once run, AiiDA calculations are instances of the class ``CalcJob``,
-more precisely of one of its subclasses, each corresponding to a code
-specific plugin (for example, the PWscf plugin). You have already seen
-``CalcJob`` classes in the previous sections.
+.. code:: python
 
-However, to create a new calculation, rather than manually creating a
-new class, the suggested way is to use a ``Builder``, that helps in
-setting the various calculation inputs and parameters, and provides
-TAB-completion.
+    code = load_code("<codename>")
+
+.. note::
 
-To obtain a new builder, we can use the ``get_builder`` method of the
-``code`` object:
+   ``load_code`` returns an object of type ``Code``, which is the general AiiDA class for describing simulation codes. 
+
+Let's build the inputs for a new ``PwCalculation`` (defined by the ``quantumespresso.pw`` plugin, the default plugin for the code you chose before)
 
 .. code:: python
 
     builder = code.get_builder()
 
-This returns a builder that helps in setting up the inputs for the
-``PwCalculation`` class (associated to the ``quantumespresso.pw``
-plugin, the default plugin for the code you chose before).
-
-As the first step, you can assign a (short) label or a (long)
-description to the calculation that you are going to create, that you
-might find convenient in the future. This can be achieved with:
+As the first step, assign a (short) label or a (long) description to your
+calculation, that you might find convenient in the future.
 
 .. code:: python
 
     builder.metadata.label = "PW test"
     builder.metadata.description = "My first AiiDA calc with Quantum ESPRESSO on Si"
 
-This information will be saved in the database for later query or
+This information will be saved in the database for later queries or
 inspection. Note that you can press TAB after writing ``builder.`` to
-see all available inputs.
+see all inputs available for this calculation.
```

Comment:
Maybe also specify here that they can do `builder.structure??` to show the auto-generated doc string from the `ProcessSpec` which will show valid types, requiredness etc. And that at any time, if they simply evaluate it, it will print the current contents

Reply:
>  if they simply evaluate it, it will print the current contents

That's not always an easy balance to strike but I would guess this is self-evident enough.